ecl/identity/v3/services/results.go

```diff
@@ -99,8 +99,8 @@ type ServicePage struct {
 }
 
 // IsEmpty returns true if the ServicePage contains no results.
-func (p ServicePage) IsEmpty() (bool, error) {
-	services, err := ExtractServices(p)
+func (r ServicePage) IsEmpty() (bool, error) {
+	services, err := ExtractServices(r)
```

Comment:
The receiver name of `ServicePage` was different in the same code, so it was unified to `r` .